cmd/ghemoji/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/donatj/ghemoji"
+)
+
+func main() {
+	// right now, we are not using flag, but we could
+	// so let's plan it
+	flag.Usage = usage
```

Comment:
We're not parsing the args right now so passing it as args does not currently work

```suggestion
	flag.Usage = usage
	flag.Parse()
```